### ISSUE TITLE
Telegram error

### ISSUE THREAD

**Issue by Prossi79** [freqtrade/freqtrade#1831]
## Step 1: Have you search for this issue before posting it?

If you have discovered a bug in the bot, please [search our issue tracker](https://github.com/freqtrade/freqtrade/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

## Step 2: Describe your environment

  * Python Version: Python 3.7.1
  * CCXT version: ccxt==1.18.468
  * Branch: Develop
  * Last Commit ID: 5f63797f17caff8f88b77d5609f9c82204e6593f

  
## Step 3: Describe the problem:

*Explain the problem you have encountered*

### Steps to reproduce:

  1. _____
  2. _____
  3. _____
  
### Observed Results:

I was running two bots in parallel on one machine, each in a seperate terminal.  Since some days the one in dry rub is not working any more, following error log: 

### Relevant code exceptions or logs:

  ```2019-05-07 13:12:14,516 - freqtrade.rpc.rpc_manager - INFO - Enabling rpc.telegram ...
2019-05-07 13:12:14,558 - freqtrade.rpc.telegram - INFO - rpc.telegram is listening for following commands: [['status'], ['profit'], ['balance'], ['start'], ['stop'], ['forcesell'], ['forcebuy'], ['performance'], ['daily'], ['count'], ['reload_conf'], ['stopbuy'], ['whitelist'], ['blacklist'], ['edge'], ['help'], ['version']]
2019-05-07 13:12:15,248 - freqtrade.exchange.exchange - INFO - Instance is running with dry_run enabled
2019-05-07 13:12:15,249 - freqtrade.exchange.exchange - INFO - Applying additional ccxt config: {'enableRateLimit': True}
2019-05-07 13:12:15,253 - freqtrade.exchange.exchange - INFO - Applying additional ccxt config: {'enableRateLimit': True, 'ratelimit': 1000}
2019-05-07 13:12:15,655 - freqtrade.exchange.exchange - INFO - Using Exchange "Binance"
2019-05-07 13:12:16,865 - freqtrade.resolvers.exchange_resolver - INFO - Using resolved exchange Binance
2019-05-07 13:12:16,866 - freqtrade.wallets - INFO - Wallets synced.
2019-05-07 13:12:16,869 - freqtrade.resolvers.pairlist_resolver - INFO - Using resolved pairlist StaticPairList from '/root/freqtrade/freqtrade/pairlist'
2019-05-07 13:12:16,987 - freqtrade.rpc.rpc_manager - INFO - Sending rpc message: {'type': status, 'status': 'config reloaded'}
2019-05-07 13:12:17,602 - freqtrade - ERROR - Fatal exception!
Traceback (most recent call last):
  File "/root/freqtrade/freqtrade/main.py", line 42, in main
    worker.run()
  File "/root/freqtrade/freqtrade/worker.py", line 72, in run
    state = self._worker(old_state=state)
  File "/root/freqtrade/freqtrade/worker.py", line 82, in _worker
    state = self.freqtrade.state
AttributeError: 'NoneType' object has no attribute 'state'
2019-05-07 13:12:20,105 - telegram.ext.updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running (409)
2019-05-07 13:12:20,106 - telegram.ext.dispatcher - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/root/freqtrade/.env/lib/python3.7/site-packages/telegram/ext/updater.py", line 319, in _network_loop_retry
    if not action_cb():
  File "/root/freqtrade/.env/lib/python3.7/site-packages/telegram/ext/updater.py", line 280, in polling_action_cb
    allowed_updates=allowed_updates)
  File "/root/freqtrade/.env/lib/python3.7/site-packages/telegram/bot.py", line 65, in decorator
    result = func(self, *args, **kwargs)
  File "/root/freqtrade/.env/lib/python3.7/site-packages/telegram/bot.py", line 1975, in get_updates
    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))
  File "/root/freqtrade/.env/lib/python3.7/site-packages/telegram/utils/request.py", line 309, in post
    headers={'Content-Type': 'application/json'})
  File "/root/freqtrade/.env/lib/python3.7/site-packages/telegram/utils/request.py", line 233, in _request_wrapper
    raise NetworkError('{0} ({1})'.format(message, resp.status))
telegram.error.NetworkError: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running (409)


  ```

**Comment by hroff-1902** [comment]
duplicated to #1810? It was fixed by #1815 a week ago. 
Pls update the bot from the current develop.

(You did not filled the fields in the submission form above ("Step 2: Describe your environment"). What's the Last Commit ID you are working with?)

**Comment by xmatthias** [comment]
Irrelevant of that (yes, please fill out these informations) - i think we should do a new release to fix this bug in master as well.

**Comment by Prossi79** [comment]
sorry, guys ... information added above

I will just update the bot
git pull origin develop
pip3 install -r requirements.txt

correct?

**Comment by Prossi79** [comment]
same problem after doing above updates ... : /

**Comment by hroff-1902** [comment]
What now is your last commit id (`git log --format="%H" -n 1`)?
Do you run the bot in the docker?
Have you updated the codebase (`git pull`) in the same directory you run the bot from (`/root/freqtrade/`)?
What's the output of `git status` now?

**Comment by hroff-1902** [comment]
You probably need to make `git checkout develop`, you may be at another branch which is different of develop.

**Comment by Prossi79** [comment]
last commit id: a70830a7b7fd708a526f6eef4cda985f20892a13
no, I dont run the bot in the docker
yes, I have updatet in the same directory you run the bot from (/root/freqtrade/)

git checkout develop:

(base) root@vmanager6003:~/freqtrade# git checkout develop
Already on 'develop'
Your branch is up to date with 'origin/develop'.

**Comment by hroff-1902** [comment]
Well, you now (right now) made checkout to the develop branch. Does the problem still persist?

**Comment by Prossi79** [comment]
okay, working now :) after I have rebooted my VPS ... maybe I had just crashed the system
